src/doctopus/template.clj

```diff
@@ -1,29 +1,29 @@
 (ns doctopus.template
   (:require [doctopus.doctopus :refer [list-heads list-tentacles]]
-   [net.cgrand.enlive-html :as enlive :refer [deftemplate defsnippet]]
+            [net.cgrand.enlive-html :as enlive :refer [deftemplate defsnippet]]
             [doctopus.configuration :refer [server-config]]
             [doctopus.doctopus :refer [list-heads list-tentacles list-tentacles-by-head]]
             [doctopus.doctopus.tentacle :refer [get-html-entrypoint]]
             [ring.util.anti-forgery :as csrf]))
 
-(defn- make-anchor
```

Comment:
turns out: you did! updated in https://github.com/Gastove/doctopus/commit/b7d69aa539967cab4b9a3f84c9394a74abc1d2b5